app/src/server/tasks/generateTestSetEntry.task.ts

```diff
@@ -166,14 +166,24 @@ export const generateTestSetEntry = defineTask<GenerateTestSetEntryJob>({
           const inputTokens = completion.usage?.prompt_tokens ?? 0;
           const outputTokens = completion.usage?.completion_tokens ?? 0;
 
+          const { cost, inputCost, outputCost } = calculateCost(
+            fineTune,
+            0,
+            inputTokens,
+            outputTokens,
+          );
+
           await prisma.usageLog.create({
             data: {
               fineTuneId: fineTune.id,
               projectId: fineTune.projectId,
+              baseModel: fineTune.baseModel,
               type: UsageType.TESTING,
               inputTokens,
               outputTokens,
-              cost: calculateCost(fineTune, 0, inputTokens, outputTokens),
+              cost,
```

Comment:
Minor point, but you could save a few lines all the places you do this by just changing it to `...calculateCost(fineTune, 0, inputTokens, outputTokens)`

Reply:
Nice hint!
I don't know which option I prefer more since it hides the actual variables, but it definitely looks succinct. Replaced it in some places.